pkg/apis/pipeline/v1/pipeline_types_test.go

```diff
@@ -969,3 +971,56 @@ func TestEmbeddedTask_IsCustomTask(t *testing.T) {
 		})
 	}
 }
+
+func TestPipelineChecksum(t *testing.T) {
+	tests := []struct {
+		name     string
+		pipeline *Pipeline
+	}{{
+		name: "pipeline ignore uid",
+		pipeline: &Pipeline{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "tekton.dev/v1",
+				Kind:       "Pipeline"},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:        "pipeline",
+				Namespace:   "pipeline-ns",
+				UID:         "abc",
+				Labels:      map[string]string{"label": "foo"},
+				Annotations: map[string]string{"foo": "bar"},
+			},
+			Spec: PipelineSpec{},
+		},
+	}, {
+		name: "pipeline ignore system annotations",
+		pipeline: &Pipeline{
+			TypeMeta: metav1.TypeMeta{
+				APIVersion: "tekton.dev/v1",
+				Kind:       "Pipeline"},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "pipeline",
+				Namespace: "pipeline-ns",
+				UID:       "abc",
+				Labels:    map[string]string{"label": "foo"},
+				Annotations: map[string]string{
+					"foo":                       "bar",
+					"kubectl-client-side-apply": "client",
+					"kubectl.kubernetes.io/last-applied-configuration": "config",
+				},
+			},
+			Spec: PipelineSpec{},
+		},
+	}}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			sha, err := tt.pipeline.Checksum()
+			if err != nil {
+				t.Fatalf("Error computing checksuum: %v", err)
```

Comment:
```suggestion
				t.Fatalf("Error computing checksum: %v", err)
```

other test files as well

Reply:
fixed.